extensions/common/iam/oauth2/oauth2-core/src/main/java/org/eclipse/dataspaceconnector/iam/oauth2/core/Oauth2Extension.java

```diff
@@ -115,7 +123,10 @@ public void initialize(ServiceExtensionContext context) {
         var privateKey = configuration.getPrivateKeyResolver().resolvePrivateKey(privateKeyAlias, PrivateKey.class);
         var tokenGenerationService = new TokenGenerationServiceImpl(privateKey);
 
-        var oauth2Service = new Oauth2ServiceImpl(configuration, tokenGenerationService, okHttpClient, jwtDecoratorRegistry, context.getTypeManager(), tokenValidationService);
+        var oauth2Service = new Oauth2ServiceImpl(configuration, tokenGenerationService, okHttpClient, jwtDecoratorRegistry,
```

Comment:
Nit: can you include on one line or separate the params on individual lines, it makes it easier to read?